api/nakadi-event-bus-api.yaml

```diff
@@ -421,16 +421,16 @@ paths:
       description: |
         Starts a stream delivery for the specified partitions of the given EventType.
 
-        The event stream is formated as a sequence of `EventStreamBatch`es separated by `\n`. Each
-        `EventStreamBatch` constains a chunk of Events and a `Cursor` pointing to the **end** of the
+        The event stream is formatted as a sequence of `EventStreamBatch`es separated by `\n`. Each
+        `EventStreamBatch` contains a chunk of Events and a `Cursor` pointing to the **end** of the
         chunk (i.e. last delivered Event). The cursor might specify the offset with the symbolic value
         `OLDEST`, which will open the stream starting from the oldest available offset in the partition.
         Currently this format is the only one supported by the system, but in the future other MIME types
-        will be suported (like `application/event-stream`).
+        will be supported (like `application/event-stream`).
 
         If streaming for several distinct partitions, each one is an independent `EventStreamBatch`.
 
-        The initialization of a stream can be parameterized in terms of size of each chunk, timeout for
+        The initialization of a stream can be parametrized in terms of size of each chunk, timeout for
```

Comment:
I think "parameterized" is actually ok:
https://dict.leo.org/ende/?lang=de&search=parameterized